pandas/tests/frame/indexing/test_insert.py

```diff
@@ -89,3 +89,10 @@ def test_insert_item_cache(self, using_array_manager):
         ser.values[0] = 99
 
         assert df.iloc[0, 0] == df[0][0]
+
+    def test_insert_frame(self):
```

Comment:
Your test is broken

Reply:
Very confused. I am getting a seperate error message on my local.

Reply:
The ci error is correct, you will have to debug locally why it is not raised for you

Reply:
changed the error msg, but this is also failing. Would need some help, how same error raises multiple error msgs